Readme.md

```diff
@@ -6,6 +6,40 @@ This repository contains the Android Java source code for a sample browser app t
 
 The MAM SDK libraries can be downloaded from [MAM SDKs and Toolkit](https://www.citrix.com/downloads/citrix-endpoint-management/product-software/mdx-toolkit.html).
 
+Getting Started
```

Comment:
Let's just include a link to the docs in developer.cloud.com.  I don't think it's a good idea to document it in multiple places, because then we'd have to keep them in sync and do twice the amount of work.

Reply:
Done